src/mouse_events.cpp

```diff
@@ -1141,4 +1165,19 @@ team & mouse_handler::current_team() {
 }
 
 mouse_handler *mouse_handler::singleton_ = nullptr;
+
+/**  Use this to prevent the mouse to be selecting any unit for
+ * highlighting its movement range. You would want to call this after
+ * highlighting [best] enemy movements in case the mouse cursor landed
+ * on a unit after the menu widget got closed. */
+void mouse_handler::prevent_units_highlight() {
```

Comment:
Please put the leading function brace on its own line.

Reply:
Will comply request.